FreeRTOS-Plus/Demo/corePKCS11_MQTT_Mutual_Auth_Windows_Simulator/main.c

```diff
@@ -97,7 +97,7 @@ static UBaseType_t ulNextRand;
 int main( void )
 {
     /***
-     * See https://www.FreeRTOS.org/mqtt_lts/index.html for configuration and usage instructions.
+     * See https://www.FreeRTOS.org/coremqtt for configuration and usage instructions.
```

Comment:
Shouldn't this be: `https://www.FreeRTOS.org/coremqtt/index.html`?

Reply:
See reply to other comments of the same.